kornia/geometry/transform/thin_plate_spline.py

```diff
@@ -3,7 +3,6 @@
 import torch
 from torch import nn
 
-from kornia.core import ones, zeros
```

Comment:
don't remove this

Reply:
https://github.com/kornia/kornia/actions/runs/7251009034/job/19752466964?pr=2690